### PR TITLE
Added --no-dockerfiles command shortcut

### PR DESCRIPTION
**Pull request by bogedal**

### CODE REVIEW

**Comment by ia3andy** [comment]
There is an easier way, I think it's just a bug that it doesn't work. I'll have a look on monday, tomorrow is holiday in France.

**Comment by bogedal** [comment]
@ia3andy Have a good one tomorrow and we'll see on monday which way you choose 👍

**Comment by bogedal** [comment]
@ia3andy From a users point of view I must admit I would argue that the --no-dockerfiles is the better solution.
I think it fits well with the two other options --no-code and --no-wrapper.
--no-code will affect beginning state of the app
--no-wrapper will affect how to build
--no-dockerfiles would affect how to run

From a readability standpoint I think --no-dockerfiles is better over -Dcodegen.no-dockerfiles, as the -D option is used for various other things like jvm-args.

File: devtools/cli/src/test/java/io/quarkus/cli/CliProjectGradleTest.java [hunk 1]
```diff
@@ -106,6 +107,8 @@ public void testCreateAppDefaults() throws Exception {
 
         Assertions.assertTrue(project.resolve("gradlew").toFile().exists(),
                 "Wrapper should exist by default");
+        Assertions.assertTrue(Files.exists(project.resolve("src/main/Docker")),
```
Comment (ebullient) [inline devtools/cli/src/test/java/io/quarkus/cli/CliProjectGradleTest.java:110]: The directory name should be `docker` (lower-case), shouldn't it? (This happens several times.. )

**Review by ebullient — DISMISSED** [review]

File: devtools/cli/src/test/java/io/quarkus/cli/CliProjectGradleTest.java (hunk 1 above)
Comment (bogedal) [reply devtools/cli/src/test/java/io/quarkus/cli/CliProjectGradleTest.java]: Yes, you are correct. I'll fix it asap

**Comment by ia3andy** [comment]
Hey @bogedal, so in the end this was working right? This PR introduces a new shortcut option name?

**Comment by bogedal** [comment]
Yes - this enables the use of the option --no-dockerfiles, which works like --no-code and --no-wrapper.

**Comment by bogedal** [comment]
> Hey @bogedal, so in the end this was working right? This PR introduces a new shortcut option name?

Yes - this enables the use of the option --no-dockerfiles, which works like --no-code and --no-wrapper.

**Review by ebullient — APPROVED** [review]

**Pull request merged**